examples/cluster/teams/platform/cluster-seed/templates/templates/team-seeds.yaml

```diff
@@ -0,0 +1,85 @@
+{{- range .Values.teams }}
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .seedNamespace }}
+spec:
+---
+apiVersion: eunomia.kohls.io/v1alpha1
+kind: GitOpsConfig
+metadata:
+  name: seed-{{ .name }}
+  namespace: {{ .seedNamespace }}
+spec:
+  serviceAccountRef: {{ .serviceAccountRef }}
+  templateProcessorImage: {{ .templateProcessorImage }}
+  resourceHandlingMode: CreateOrMerge
+  resourceDeletionMode: Delete
+  triggers:
+  - type: Change
+  - type: Periodic
+    cron: {{ .triggers.cron | quote }}
+  templateSource:
+{{- if $.Values.overwrite.uri }}
+    uri: {{ $.Values.overwrite.uri }}
+{{- else }}
+    uri: {{ .templateSource.uri }}
+{{- end }}
+{{- if $.Values.overwrite.ref }}
+    ref: {{ $.Values.overwrite.ref }}
+{{- else }}
+    ref: {{ .templateSource.ref }}
+{{- end }}
+    contextDir: {{ .templateSource.contextDir }}
+    #secretName: {{ .parameterSource.secretName }}
+  parameterSource:
+{{- if $.Values.overwrite.uri }}
+    uri: {{ $.Values.overwrite.uri }}
+{{- else }}
+    uri: {{ .parameterSource.uri }}
+{{- end }}
+{{- if $.Values.overwrite.ref }}
+    ref: {{ $.Values.overwrite.ref }}
+{{- else }}
+    ref: {{ .parameterSource.ref }}
+{{- end }}
+    contextDir: {{ .parameterSource.contextDir }}
+    #secretName: {{ .parameterSource.secretName }}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .serviceAccountRef }}
+  namespace: {{ .seedNamespace }}
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ .serviceAccountRef }}
+  namespace: {{ .seedNamespace }}
+subjects: 
+- kind: ServiceAccount
+  name: {{ .serviceAccountRef }}
+  namespace: {{ .seedNamespace }}
+roleRef:
+  kind: ClusterRole
+  name: admin
+  apiGroup: rbac.authorization.k8s.io  
+---
+# We need to remove this once issue #53 is figured out
```

Comment:
@Smiley73 is checking if this is needed. Requesting changes until it is figured out